identity-credential/src/credential/bitmap_revocation_status.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+
+use identity_core::common::Object;
+use identity_core::common::Url;
+use identity_core::common::Value;
+use identity_did::did::DIDUrl;
+use identity_did::did::DID;
+use identity_did::service::BITMAP_REVOCATION_METHOD;
+
+use super::Status;
+use crate::error::Error;
+use crate::error::Result;
+
+const REVOCATION_LIST_INDEX: &str = "revocationListIndex";
```

Comment:
Suggestion: rename to `revocationBitmapIndex`?

```suggestion

const INDEX_PROPERTY_NAME: &str = "revocationBitmapIndex";
```